node/internal.ts

```diff
@@ -442,10 +442,10 @@ function _tryGetExecutablePath(filePath: string, extensions: string[]): string {
                 }
             }
             else {
-                // on Mac/Linux, test the execute bit
+                // on Mac/Linux, test the execute bits
                 //     R   W  X  R  W X R W X
                 //   256 128 64 32 16 8 4 2 1
-                if ((stats.mode & 1) == 1) {
+                if ((stats.mode & 73) > 0) {
```

Comment:
This approach seems OK to me, although it's technically inaccurate (slightly - e.g. am i the owner, group member, or everyone?). It might be worth asking Zach if he has any quick ideas.

I'm curious in the case where it's inaccurate, what's the result? How does the error manifest to the user? I want to avoid a future investigation for an error surfacing in a cryptic way.